### ISSUE TITLE
tdict.get() function calls `raise default`.

### ISSUE THREAD

**Issue by rindis** [noahbenson/pcollections#3]
The tdict `get(...)` function calls `raise default`, when the intention is probably to `return default`

**Comment by rindis** [comment]
```python
    def get(self, key: K, default: Optional[V]=None) -> Optional[V]:
        h = hash(key)
        ii = self._idx.get(h, None)
        while ii is not None:
            (kv,ii) = self._els[ii]
            if key == kv[0]:
                return kv[1]
        raise default   # <- This should probably be 'return default'
```

**Comment by noahbenson** [comment]
Good catch! Fixed by commit 403ae55.
